src/dsync/dsync.c

```diff
@@ -2945,6 +2947,12 @@ int main(int argc, char **argv)
         case 'P':
             mfu_progress_timeout = atoi(optarg);
             break;
+        case 'X':
+            copy_opts->preserve_xattrs = true;
+            if(rank == 0) {
+                MFU_LOG(MFU_LOG_INFO, "Preserving extended attributes.");
+            }
+            break;
```

Comment:
Same as for `dcp`; it's okay to repurpose `X` here, but Ithink we need to use _**some**_ other option for `--daos-prefix`